spec/factories/regions.rb

```diff
@@ -0,0 +1,24 @@
+# == Schema Information
+#
+# Table name: regions
+#
+#  id                                         :uuid             not null, primary key
+#  claims_funding_available_per_hour_currency :string           default("GBP"), not null
+#  claims_funding_available_per_hour_pence    :integer          default(0), not null
+#  name                                       :string           not null
+#  created_at                                 :datetime         not null
+#  updated_at                                 :datetime         not null
+#
+FactoryBot.define do
+  factory :region do
+    name { ["Inner London", "Outer London", "Fringe", "Rest of England"].sample }
+    claims_funding_available_per_hour do
+      {
+        "Inner London" => 53.60,
+        "Outer London" => 48.25,
+        "Fringe" => 45.10,
+        "Rest of England" => 43.80,
+      }[name]
```

Comment:
This conversion is implicit, but I wonder if it is better to be explicit, so no one makes the assumption that the data is being saved as a float/decimal? What do you think?

```suggestion
    claims_funding_available_per_hour do
      {
        "Inner London" => Money.from_amount(53.60, "GBP"),
        "Outer London" => Money.from_amount(48.25, "GBP"),
        "Fringe" => Money.from_amount(45.10, "GBP"),
        "Rest of England" => Money.from_amount(43.80, "GBP"),
      }[name]
```

You can also use the money shorthand for GBP, `Money.gbp(53.60)`.